Google-Summer-of-Code-2023.md

```diff
@@ -0,0 +1,272 @@
+
+A: You should have at least 2 PRs merged, but they don't need to be large. Beyond that, remember that quality is more important than quantity. It is better to submit a non-trivial PR rather than a one-line wording change. Start with starter issues, then move ahead with prioritized issues from the team that is offering the project you are interested in.
+Here are the boards of the respective teams: [LaCE Quality](https://github.com/orgs/oppia/projects/3/views/8), [Developer Workflow](https://github.com/orgs/oppia/projects/8/views/11), [Contributor Experience](https://github.com/orgs/oppia/projects/18/views/4).
+
+**Q: What is the total number of contributors that will be accepted?**
+
+A: As many as we think will succeed, though the Google GSoC admins may impose limits based on how they decide to distribute contributor slots among the different open-source organizations.
+
+**Q: I do not have any experience in skill XYZ. Is some certification required?**
+
+A: Try to work on good first issues and take courses online. In the field of software development, it is common to develop experience and expertise as you take up and complete projects successfully. We do not require any formal certification of particular skills.
+
+**Q: Is it okay if I only focus on the frontend or backend?**
+
+A: This probably depends on the project(s) you wish to apply for. However, note that the ability to be effective in both the frontend and backend will open up more opportunities for you, since projects often touch multiple layers of the stack.
+
+**Q: The [Google GSoC FAQ](https://developers.google.com/open-source/gsoc/faq#can_someone_already_participating_in_open_source_be_a_gsoc_contributor) mentions that the program is only for new contributors. I have already contributed to Oppia and I have write access. Can I still participate?**
+
+A: The GSoC program is open to new and beginner contributors to open source and to students. Some of our contributors with write access are still beginner contributors, whereas some of our other contributors with write access will not qualify because they are experienced contributors. If you have only recently received write access, or have been contributing to Oppia for less than a year, you are probably still a beginner contributor. If the previous sentence does not apply to you, and you want to know which group you fall into, please contact **@srijanreddy98**, @U8NWXD or **@aasiffaizal** for a decision.
+
+## Dates and Deadlines
+
+Noteworthy dates for 2023 ([Full Timeline](https://developers.google.com/open-source/gsoc/timeline)):
+
+- **Jan 23 - Feb 7**: Mentoring organizations apply
+- **Feb 22**: Mentoring organizations are announced
+- **Mar 20 - Apr 4**: GSoC contributor application period
+- **May 4**: Accepted GSoC contributors are announced
+- **May 4 - May 28**: Community bonding period
+- **May 29 - Jul 14**: Milestone 1 work period for GSoC contributors
+- **Jul 15 - Jul 16**: External evaluation for M1
```

Comment:
I wonder if we should call this the "internal evaluation" since the GSoC one is kind of the external one?

Reply:
Well, yeah. Internal evaluation makes more sense